src/css/styles.css

```diff
@@ -361,9 +391,11 @@ iframe {
 }
 
 .loading {
+  align-items: center;
+  background: transparent;
   display: flex;
   justify-content: center;
-  align-items: center;
+  margin: 55px 0 5px;
```

Comment:
This works *ok* for now where we don’t render the loading indicator in too many places, but there are a lot of different places where we could/should, and I’m not sure the 55px margin here will always be appropriate. Even some places now are misaligned, e.g. the initial page load before showing anything is now off center:

<img width="500" alt="screen shot 2017-11-23 at 11 58 31 am" src="https://user-images.githubusercontent.com/74178/33187337-f45c682e-d045-11e7-956e-339702d26aef.png">

I’m assuming this is because of the fixed header? Maybe this rule should be more contextual, e.g. on `#application > .loading` or `.navbar + .loading` or something along those lines.

Reply:
Yep, it's because of the fixed header. I added a contextual rule as you suggested. Works well!

Reply:
Cool. I think we may as well go ahead and merge, then. For future work along these lines, did you look into `position: sticky` at all? It should actually impact the available space (so you don’t need to add the margins), unlike `position: fixed`.

Reply:
Embarrassingly, I didn't know that was a new option. That's cool that it remains in the document flow. I tested it out and think I got it working properly. I'll put it in a new PR.